doc/install.rst

```diff
@@ -63,6 +63,16 @@ softwares should also be installed.
   
   sudo pip install docker-py
 
+To support :ref:`Hosts Pool <yorc_infras_hostspool_section>` with SSH password authentication instead of common ssh keys, you need
```

Comment:
Isn't it applicable to all infrastructures using ansible ?

Reply:
yes